src/local.rs

```diff
@@ -192,118 +201,107 @@ impl LocalFileSystem {
 
         Ok(Path::parse(decoded)?)
     }
-
-    async fn get_file(&self, location: &Path) -> Result<(fs::File, std::path::PathBuf)> {
```

Comment:
This is moved to not be a member function, to allow it to be called without a reference to self